src/clp_logging/handlers.py

```diff
@@ -705,19 +717,24 @@ def __init__(
             loglevel_timeout.set_ostream(self.ostream)
         self.loglevel_timeout = loglevel_timeout
 
+    def _encode_log_event(self, msg: str) -> bytearray:
+        timestamp_ms: int = floor(time.time() * 1000)
+        clp_msg: bytearray = FourByteEncoder.encode_message_and_timestamp_delta(
+            timestamp_ms - self.last_timestamp_ms, (msg + "\n").encode()
+        )
+        self.last_timestamp_ms = timestamp_ms
+        return clp_msg
```

Comment:
This helper is nice. Generalize it so that you can also replace the same blocks found in `_aggregator` (and `log_fn` inside it).